tests/benchmarks/management_api_b.cpp

```diff
@@ -74,7 +74,7 @@ static void BenchmarkHasPolicyMedium(benchmark::State& state) {
         params = { "user" + std::to_string(GetRandom1000()), "data" + std::to_string(GetRandom1000()/10), "read" }, e.HasPolicy(params);
 }
 
-BENCHMARK(BenchmarkHasPolicyMedium);
+// BENCHMARK(BenchmarkHasPolicyMedium);
```

Comment:
Maybe we need a `macro` to cover those code consume too much time, and makes `benchmark` is easy to test for users. What do you think? @noob20000405

Reply:
@EmperorYP7

Reply:
We may also add a CMake option named `INTENSIVE_BENCHMARK` defaulting it to `OFF`. The user may turn it on through `-DINTENSIVE_BENCHMARK=ON` while generating the project files.

Reply:
@noob20000405 can you add it?

Reply:
Sure! I'll do that right away.

Reply:
> @noob20000405 can you add it?

Sure! I'll do that right away.